src/app/navbar/navbar.component.html

```diff
@@ -30,7 +30,7 @@
           <img class="site-icons-small hidden-sm" src="../assets/images/dockstore/dockstore-documentation.png" alt="stacked pages" /> Docs
         </a>
         <a mat-button class="icon" [href]="Dockstore.DISCOURSE_URL" target="_blank" rel="noopener noreferrer">
-          <mat-icon class="hidden-sm">help_outline</mat-icon> Help
+          <mat-icon class="hidden-sm">forum</mat-icon> Forum
```

Comment:
What is this change? I don't see the difference in the screenshot.

Reply:
> What is this change? I don't see the difference in the screenshot.

Ah, I had only looked at the first screen shot, in the PR description. Now I see it.